templates/report.html

```diff
@@ -0,0 +1,319 @@
+{% extends "base.html" %}
+
+{% block title %}{{title}}{% endblock %}
+{% block description %}{{description}}{% endblock %}
+{% block type %}article{% endblock %}
+{% block img %}https://{{SITE_DOMAIN}}{{img}}{% endblock %}
+{% block subtitle %}<a href="{{PAGES.blog.url}}">Blog</a>{% endblock %}
```

Comment:
may this link to `/reports` instead `/blog`?

Reply:
I'm thinking to list all the publications we do (i.e blog posts and reports) on place, right now that's the `/blog` page. Perhaps we can rename that page to something more generic, like 'Insights' or 'Stories', but would need to discuss with the bizdev team first. Anyway, I think for now we can leave this as-is and then decide on names and organization with the rest of the team